### ISSUE TITLE
definition of the interface fluxes for subdomain p+

### ISSUE THREAD

**Issue by reyhanhgh** [AmeyaJagtap/Conservative_PINNs#2]
Dear Professor Jagtap,

In line 290 of your code we have: `fi1 = u1i1**2/2-self.nu*u1i1_x - (u2i1**2/2-self.nu*u2i1_xx)`. Taking into account the 1D Burgers equation under the conservative forme, we have `u_t+f_x=0` where `f` is `u^2/2 - nu * u_x`. This definition of `f` is consistent with `u1i1**2/2-self.nu*u1i1_x` defined as the interface flux of the subdomain 1 given by NN1. My question is that why do you use `u2i1_xx` and not `u2i1_x` for the adjacent subdomain (NN2)? I would be grateful if you could explain this matter.

Thank you.

**Comment by charlincc** [comment]
Yes,I have got the same problem. I think the result shoulde be " self.nu*u1i1_x ".
